src/seedu/addressbook/AddressBook.java

```diff
@@ -486,16 +503,16 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<HashMap<Str
      */
     private static ArrayList<HashMap<String, String>> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
         final ArrayList<HashMap<String, String>> matchedPersons = new ArrayList<>();
+		final Set<String> keyWordsInNameInUpperCase = changeWordsInNameToUpperCase(new HashSet<String>(keywords));
         for (HashMap<String, String> person : getAllPersonsInAddressBook()) {
             final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
             final Set<String> wordsInNameInLowerCase = changeWordsInNameToLowerCase(wordsInName);
-			if (!Collections.disjoint(wordsInName, keywords)) {
-                matchedPersons.add(person);
-            }
-			
+			final Set<String> wordsInNameInUpperCase = changeWordsInNameToUpperCase(wordsInName);
 			if (!Collections.disjoint(wordsInNameInLowerCase, keywords)) {
                 matchedPersons.add(person);
-            }
+            } else if (!Collections.disjoint(wordsInNameInUpperCase, keyWordsInNameInUpperCase)){
+				matchedPersons.add(person);
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.